pkgs/development/tools/build-managers/bazel/bazel_7/default.nix

```diff
@@ -574,11 +696,13 @@ stdenv.mkDerivation rec {
     #     nix-build . -A bazel_7.tests
     #
     # in the nixpkgs checkout root to exercise them locally.
-    tests = callPackage ./tests.nix {
-      inherit Foundation bazel_self lockfile repoCache;
-    };
+    # tests = callPackage ./tests.nix {
+    #   inherit Foundation bazel_self lockfile repoCache;
+    # };
+    # TODO tests have not been updated yet and will likely need a rewrite
+    # tests = callPackage ./tests.nix { inherit Foundation bazel_deps bazel_self; };
 
     # For ease of debugging
-    inherit distDir repoCache lockfile;
+    inherit bazel_deps bazel_fhs;
```

Comment:
Should these be in `passthru`, or what's the catch here?

Reply:
Despite being totally different as a result of the vendoring change, these were also useful for debugging (though I wonder if these should be camelCase instead), so that's why I included them.